### ISSUE TITLE
Enable dev environment on Apple Silicon computers

### ISSUE THREAD

**Issue by eidens** [source-data/sd-graph#9]
The current dev environment can't be set up on Macbooks running the new M1 processor. Running the "First time setup" commands described in README.md (`docker-compose build` and `docker-compose up`) results in this error:

```
ERROR: Could not find a version that satisfies the requirement tensorflow==1.14.0 (from smtag) (from versions: none)
ERROR: No matching distribution found for tensorflow==1.14.0
```

The error occurs because tensorflow doesn't provide ARM binaries as of yet: ["Apple Silicon support is provided via a fork from Apple, At the moment there is no way TF team at Google can maintain that build."](https://github.com/tensorflow/tensorflow/issues/45631#issuecomment-751412972)

**Comment by eidens** [comment]
Docker (to be precise, "Docker Desktop for Mac on Apple Silicon") can build and run x86 images on an M1 computer using Apple's Rosetta: https://docs.docker.com/desktop/mac/apple-silicon/

This can be used by either passing the [`--platform`](https://docs.docker.com/engine/reference/commandline/build/#options) flag to `docker build` or adding the [configuration parameter of the same name](https://docs.docker.com/compose/compose-file/compose-file-v2/#platform) to each service in the docker-compose file.

**Comment by eidens** [comment]
Fixed by #10.
